verilog/formatting/tree_unwrapper.cc

```diff
@@ -1294,6 +1336,19 @@ void TreeUnwrapper::ReshapeTokenPartitions(
     case NodeEnum::kStatement: {
       // This handles cases like macro-calls followed by a semicolon.
       AttachTrailingSemicolonToPreviousPartition(&partition);
+
+      const auto* subtree = GetSubtreeAsSymbol(node, NodeEnum::kStatement, 0);
```

Comment:
Function calls in general can appear in expressions and statements.  Does it make sense to handle this partition reshaping in a case outside of `kStatement`?